### ISSUE TITLE
[Feature Request] Custom commands on selectfile event

### ISSUE THREAD

**Issue by tupo2** [donmaiq/Mpv-Playlistmanager#119]
Hi JoJo,

Instead of just move the selected file in the playlist, would it be possible to add an event, for example when the selected file is reached in the playlist?
Example, pause playing on selected file:
```
KEY script-message playlistmanager onselectfile <command>
KEY script-message playlistmanager onselectfile set pause yes
```

**Comment by jonniek** [comment]
This seems very niche. It's not impossible, but also it seems kind of awkward to use the selection for this. There might be some possibility to make it a bit more general. Something like:

```
KEY script-message playlistmanager event cursor onload <command>
KEY script-message playlistmanager event cursor onclose <command>
```

Something like this maybe :thinking: So if you run the `event cursor onload pause yes` script message it will register the command to be run when the hovered file is loaded. This would allow setting up multiple of such events rather than locking the selection for it.

Still need to think a bit if this belongs in this script or if it's possible to make a separate one.

**Comment by tupo2** [comment]
Yes, using the cursor seem a better solution than the selection. A command to lock the cursor position would be handy too, to avoid accidentally moving it.

**Comment by CogentRedTester** [comment]
@jonniek my [custom keybind](https://github.com/CogentRedTester/mpv-file-browser/blob/master/custom-keybinds.md) from file-browser might be interesting to you if you're thinking of going down this route. You'd probably want a much simpler system though.

**Comment by jonniek** [comment]
> A command to lock the cursor position would be handy too, to avoid accidentally moving it.

You can change 
```
sync_cursor_on_load
reset_cursor_on_close
reset_cursor_on_open
```
in settings to "no" for permanent change or at runtime with `KEY change-list script-opts append playlistmanager-sync_cursor_on_load=no`

**Comment by tupo2** [comment]
When I say _lock the cursor_, I mean to not allow the cursor to move up and down.
